gearmand/datadog_checks/gearmand/gearmand.py

```diff
@@ -105,7 +105,7 @@ def check(self, instance):
         client = self._get_client(host, port)
         self.log.debug("Connected to gearman")
 
-        tags += service_check_tags
+        tags = tags + service_check_tags
```

Comment:
This is a bug. `tags += service_check_tags` will modify the initial list of tags that belong to the instance object.

Reply:
Nice catch. I would change the logic though, because the tags are not just used for service checks:
```
check_tags = ["server:{}".format(host), "port:{}".format(port)]
check_tags.extend(tags)
```